README.md

```diff
@@ -22,25 +22,38 @@ ___
 
 ## Implementation
 
-### Stratis ID URI
+### Stratis ID
 
-A Stratis ID URI consists of three required parts, as well as optional parts:
+A Stratis ID consists of the following parts:
 
-- The scheme - `sid`
-- The callback - a protocol-relative URL which the wallet will send a HTTPS request
+- The callback - a protocol-relative URL to which the wallet will send a HTTPS request
 - The UID - a unique identifier for a request, which is present within the query string of the callback
 - The expiry datetime (optional) - a unix timestamp that specifies when the signature should expire, which is present within the query string of the callback
 
-**Example:** `sid:api.example.com/auth?uid=4606287adc774829ab643816a021efbf&exp=1647216000`
+**Example:** `api.example.com/auth?uid=4606287adc774829ab643816a021efbf&exp=1647216000`
+
+#### Handling the Stratis ID URI
+
+You can handle the Stratis ID in two ways:
+
+1. As a URI, using the scheme _sid:_
+    ```
+    sid:api.example.com/auth?uid=4606287adc774829ab643816a021efbf&exp=1647216000
+    ```
+2. With a protocol handler URI, using the scheme _web+sid:_
+    ```
+    web+sid:api.example.com/auth?uid=4606287adc774829ab643816a021efbf&exp=1647216000
```

Comment:
Slight tweak, rather than `web+sid:api.example...` it would need to be in URL format like `web+sid://api.example...`. 

dApps would need to include a button directly linking to the web+sid URL which would prompt the browser to check and ask which extension to use. 

Clients probably should have set extensions that they check for prior to showing web+sid options but I don't personally think UI decisions should in included in the spec as something apps _need_ to do.

Reply:
Tried it out with the new pre-release of the Blockcore extension and using the `web+sid:api.example.com` format works. The authority part isn't required by chrome, or safari (using safari opens the chrome extension). Therefore I believe it best we keep this format, as it adheres to the URI spec.